src/App.js

```diff
@@ -1,25 +1,50 @@
 import React from 'react';
+import { Clock } from './Components/Clock';
 
 import './App.scss';
 
-const App = () => {
-  setInterval(() => {
-    const date = new Date();
+class App extends React.Component {
+  state = {
+    clockVisible: true,
+    clockName: 1,
+  }
 
-    // eslint-disable-next-line
-    console.log(date.toLocaleTimeString());
-  }, 1000);
+  render() {
+    return (
+      <>
+        <div className="App">
+          <h1>React clock</h1>
+          <p>
+            Current time:
+            {' '}
+            {this.state.clockVisible
+            && <Clock clockName={this.state.clockName} /> }
+          </p>
+          <button
+            type="button"
+            onClick={() => this.setState({ clockVisible: false })}
```

Comment:
better to create separate methods and call them onClick

Reply:
Done